android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -48,6 +48,8 @@ public final class ExoPlayerView extends FrameLayout implements AdViewProvider {
     private @ViewType.ViewType int viewType = ViewType.VIEW_TYPE_SURFACE;
     private boolean hideShutterView = false;
 
+    private SubtitleStyle localStyle = SubtitleStyle.parse(null);
```

Comment:
I think it would make more sense to initialize `localStyle` in the constructor. We can still define the variable outside of it, but by initializing it in the constructor, we ensure everything is properly set up from the start. This approach usually leads to clearer and more reliable code. WDYT?

Reply:
You are right, it makes more sense ! This is fixed now